ogr/ogrfeature.cpp

```diff
@@ -1590,6 +1590,61 @@ int OGRFeature::IsFieldSet(int iField) const
     }
 }
 
+/************************************************************************/
```

Comment:
Is it really useful to have a method at OGRFeature level that is a small convenience layer on top of the OGRFieldDefn API ? and in the C API in particular. We don't do this for other properties of OGRFieldDefn

Reply:
I don't know, I've got the inspiration from OGR_F_IsFieldNull and OGR_F_IsFieldSetAndNotNull. I can remove them if you prefer.

Reply:
IsFieldNull is used extensively, hence a rather justified need for a shortcut helper. My slight preference would be to not increase the API surface for IsGenerated/SetGenerated at the OGRFeature level as it is likely going to be of marginal use